cypress/cypress-smoketests/integration/verify-email-receipt.cy.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  changeEmailTo,
+  clickChangeEmail,
+  clickConfirmModal,
+  confirmEmailPendingAlert,
+  goToMyAccount,
+} from '../../cypress-integration/support/pages/my-account';
+import { navigateTo } from '../../cypress-integration/support/pages/maintenance';
+
+if (!Cypress.env('SMOKETESTS_LOCAL') && !Cypress.env('MIGRATE')) {
+  describe('Verify verification email', () => {
+    const bucketName = Cypress.env('SMOKETEST_BUCKET');
+    const emailDomain = Cypress.env('EFCMS_DOMAIN');
+    const uniqueTimestamp = Date.now();
+    const testEmailAddress = `smoketest+${uniqueTimestamp}@${emailDomain}`;
+
+    before(() => {
+      cy.task('deleteAllItemsInEmailBucket', bucketName);
+    });
+
+    after(() => {
+      cy.task('deleteAllItemsInEmailBucket', bucketName);
+    });
+
+    it('should update petitioner email and confirm that a verification email is received by the updated email address', () => {
+      navigateTo('petitioner9');
+      goToMyAccount();
+      clickChangeEmail();
+      changeEmailTo(testEmailAddress);
+      clickConfirmModal();
+      confirmEmailPendingAlert();
+
+      readS3InboxAndRetry({ bucketName, testEmailAddress });
+    });
+  });
+} else {
+  describe('we do not want this test to run locally, so we mock out a test to make it skip', () => {
+    it('should skip', () => {
+      expect(true).to.equal(true);
+    });
+  });
+}
+
+const readS3InboxAndRetry = ({
```

Comment:
Can we use the existing retry method Cody created?

Reply:
We did initially refactor to use Cody's retry, but then realized we don't need to be retrying in Cypress and moved the logic to the `read` and `delete` tasks

Reply:
Loved that this moved into the task! nice move.